common/src/leap/soledad/common/couch.py

```diff
@@ -476,9 +434,14 @@ def __init__(self, url, dbname, replica_uid=None, ensure_ddocs=True):
             self._set_replica_uid(replica_uid)
         if ensure_ddocs:
             self.ensure_ddocs_on_db()
-        # initialize a thread pool for parallelizing get_docs()
-        self._sem_pool = threading.BoundedSemaphore(
-            value=self.MAX_GET_DOCS_THREADS)
+        self._cache = None
```

Comment:
Why isn't the cache initialized here?

I see it's initialized by `leap.soledad.server.sync`. Looks like the cache name depends on the `sync_id` (I don't understand yet why), could we pass it as parameter of CouchdbDatabase.__init__? Or is there any case where we don't want to have a cache?